input/pagecontent/changes.md

```diff
@@ -7,4 +7,9 @@ This change log documents the significant updates and resolutions implemented fr
 
 #### Changes in this version
 
-TBD
\ No newline at end of file
+- Removed Must Support from the following elements in AU Core Procedure:
+  - Procedure.statusReason [FHIR-45013](https://jira.hl7.org/browse/FHIR-45013)
+  - Procedure.category [FHIR-45014](https://jira.hl7.org/browse/FHIR-45014)
+  - Procedure.recorder [FHIR-45015](https://jira.hl7.org/browse/FHIR-45015)
+  - Procedure.asserter [FHIR-45016](https://jira.hl7.org/browse/FHIR-45016)
+  - Procedure.bodySite [FHIR-45017](https://jira.hl7.org/browse/FHIR-45017)
```

Comment:
think this point on bodySite should also state that added profile specific implementation guidance on including body site information